packages/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -12,13 +16,13 @@ export const ProgressBar = forwardRef<
   const widthPercentage = ((props.value ?? 0) / (props.max ?? 100)) * 100;
 
   return (
-    <Box asChild border="1" h="6" overflow="hidden" w="1/2">
+    <Box asChild border="1" h="6" overflow="hidden" {...props}>
```

Comment:
@ImperishableMe this is what I got from Figma:

<img width="377" alt="image" src="https://github.com/optimizely-axiom/optiaxiom/assets/8050659/e2011989-47a9-49e9-b68f-20605d1d6047">

<img width="173" alt="image" src="https://github.com/optimizely-axiom/optiaxiom/assets/8050659/d4d31d21-709e-4d54-a52a-afcb8fa8d1e7">

The whole thing is rounded.

Reply:
And there's no border. Did we intentionally decide to add border?

cc: @radnan